### ISSUE TITLE
UnambiguousTimeoutError: unambiguous timeout

### ISSUE THREAD

**Issue by ryanholder** [couchbase-examples/nextjs-quickstart#44]
I am having an issue when I try to run the example application against a Couchbase Capella cluster. I am running a trial and have configured the app to run on Vercel as per post and Youtube video.

No matter what I do I get the following error;
```
Server Error
UnambiguousTimeoutError: unambiguous timeout

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
errorFromCpp
```
I have even tried running it via Gitpod and updating the env file to point to Capella instance.

Any advice to what I may be doing wrong would be greatly appreaciated.

**Comment by ejscribner** [comment]
Hey @ryanholder, thanks for raising this issue! 

It seems that changes in the Couchbase Node.js SDK version 4.0.0 are interfering with this project connecting to cloud-based clusters. I'll look into this issue further for a more permanent solution, but in the mean time I think downgrading to 3.2.4 will fix your issue. 

In package.json under dependencies: 
`"couchbase": "3.2.4",`

Then be sure to run `npm install`.

Also, be sure to set the `IS_CLOUD_INSTANCE` variable in `.env.local` to `true`. 

Currently, the bucket and collection initialization code is only set up for locally hosted clusters so you'll have to manually create a bucket called `user_profile` and then within that buckets default scope, a collection called `profile`. [See here for more info on managing buckets in Capella.](https://docs.couchbase.com/cloud/clusters/data-service/manage-buckets.html)

After creating the bucket and collection, you can still run `npm run init-db:local` to create the required indices. The bucket and collection creation steps will fail with `ECONNREFUSED` but it should still be able to create the index on your Capella cluster, if `.env.local` is configured correctly. 

I hope this helps, but feel free to follow up if issues continue! I'll be looking for a more permanent solution, and will update the README and guides appropriately.

**Comment by ejscribner** [comment]
Great, glad to hear! Will look for a better solution and update the project accordingly, but glad you've got it working for the time being.

**Comment by ejscribner** [comment]
@ryanholder wanted to follow up on this issue, as I've discovered a more permanent solution. 

The connection string parameter `?ssl=no_verify` was deprecated in the latest SDK release, and was replaced with `?tls_verify=none`. Simply replacing ?ssl=no_verify` with the new parameter should solve the `UnambiguousTimeoutError` and allow you to use the Couchbase Node.js 4.0.0 SDK. 

The example code has also been updated.
